snuba/datasets/storages/generic_metrics.py

```diff
@@ -48,9 +48,11 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
     """
     return ProduceInvalidMessagePolicy(
         KafkaProducer(
-            build_kafka_producer_configuration(Topic.DEAD_LETTER_GENERIC_METRICS)
+            build_kafka_producer_configuration(
+                Topic.DEAD_LETTER_GSNUBA_DEAD_LETTER_GENERIC_METRICSENERIC_METRICS
```

Comment:
Apologies if this has been addressed already, but is the Topic here on line 52 a valid topic? While working on testing sliced consumers, this threw an Attribute error as I think it is not in `_HARDCODED_TOPICS` or `_REGISTERED_TOPICS`.

Reply:
Good catch, that's definitely a copy paste mistake. Updated!